src/renderer/components/query-result.jsx

```diff
@@ -75,14 +75,26 @@ export default class QueryResult extends Component {
       const title = fields[0].name;
       return (
         <Message
-          key={queryIndex}
+          key={`explain-${queryIndex}`}
           preformatted
           title={title}
           message={rows.map((row) => row[title]).join('\n')}
         />
       );
     }
 
+    // Not sure what type of query they ran, but cannot render table, print
+    // generic message.
+    if (fields.length === 0) {
+      return (
+        <Message
+          key={`genericResult-${queryIndex}`}
+          message={`Query executed successfully.`}
+          type="success"
+        />
+      );
+    }
```

Comment:
I put this `if` statement here (as opposed to high-up as possible) to act as a final catch-all before rendering the table, and that we may add to the the possible conditions in which we enter this if statement, and do not want to worry about colliding with `explain` or anything else we might put above it,